fk/ui.go

```diff
@@ -51,3 +52,17 @@ func printFailedAction(message string) {
 func printHeader(message string) {
 	out.Print(colour.Header(fmt.Sprintf(" %-79s", message)) + "\n\n")
 }
+
+func printFileDivider(message string) {
```

Comment:
It would be awesome to add the example from the commit message to a comment on this function. From the code I struggled to work out what it's for!
e.g.
```
// printFileDivider prints a divider string, including a message if given, e.g.:
// ── readme.txt ──────────────────────────────────────────────────
```

Reply:
I've just pushed a commit to add this!